src/bzflag/DisplayMenu.cxx

```diff
@@ -134,44 +136,86 @@ DisplayMenu::DisplayMenu() : formatMenu(NULL)
 
     option = new HUDuiList;
     option->setFontFace(fontFace);
-    option->setLabel("Quality:");
-    option->setCallback(callback, "6");
+    option->setLabel("AntiFlicker:");
+    option->setCallback(callback, "R");
     options = &option->getList();
-    options->push_back(std::string("Low"));
-    options->push_back(std::string("Medium"));
-    options->push_back(std::string("High"));
-    options->push_back(std::string("Experimental"));
+    options->push_back(std::string("Off"));
+    options->push_back(std::string("On"));
     option->update();
     listHUD.push_back(option);
 
+
+
     option = new HUDuiList;
     option->setFontFace(fontFace);
-    option->setLabel("Shadows:");
-    option->setCallback(callback, "7");
+    option->setLabel("Dithering:");
+    option->setCallback(callback, "1");
     options = &option->getList();
     options->push_back(std::string("Off"));
-    options->push_back(std::string("Stipple"));
-    options->push_back(std::string("Stencil"));
+    options->push_back(std::string("On"));
     option->update();
     listHUD.push_back(option);
 
-#if defined(DEBUG_RENDERING)
     option = new HUDuiList;
     option->setFontFace(fontFace);
-    option->setLabel("Hidden Line:");
-    option->setCallback(callback, "a");
+    option->setLabel("Blending:");
+    option->setCallback(callback, "2");
     options = &option->getList();
     options->push_back(std::string("Off"));
     options->push_back(std::string("On"));
     option->update();
     listHUD.push_back(option);
 
+    option = new HUDuiList;
+    option->setFontFace(fontFace);
+    option->setLabel("Smoothing:");
+    option->setCallback(callback, "3");
+    options = &option->getList();
+    options->push_back(std::string("Off"));
+    options->push_back(std::string("On"));
+    option->update();
+    listHUD.push_back(option);
+
+
+
+    if (((BzfWindow*)getMainWindow()->getWindow())->hasGammaControl())
```

Comment:
C-style casts are evil. If you must cast, prefer "ugly" C++ casts (better to expose the proper interfaces on high-level objects)
```suggestion
    if ( dynamic_cast<BzfWindow*>(getMainWindow()->getWindow())->hasGammaControl())
```

Reply:
Is there even a reason we are casting at all?  MainWindow::getWindow() already returns a BzfWindow*.